### DISCUSSION TITLE
Terminal/Logbook Improvements

### DISCUSSION THREAD

**Discussion by ven1883** [FakeFishGames/Barotrauma#13784] [Ideas and suggestions]
I was actually really disappointed when I got back into playing with the terminals. They're simultaneously extremely powerful and also some of the weakest components I can think of.

Some grievances:

- Welcome Text is destroyed on clearing the terminal. This wouldn't be that big of a deal if it was a toggleable option. I would love to be able to have it reappear every time so I can actually fill out the terminal with decor.
- Text length limit. Why does this exist, and why is it so short? The Welcome Text isn't limited, but typing or inputting text through wires is. Because of this, I have to fanangle a chain of sigchecks and delays to dump the characters I want into the terminal.

Honestly, it's not that big of a deal, but please, it shouldn't be that hard to change something like the text length limit.
Sub editors worldwide beg of you.

**Comment by NotWendy** [comment]
It would be nice to have the option to make the welcome text reappear when the terminal's contents are cleared. Most terminal circuits do it with only two additional components, of course. But it'd be convenient to have as a built-in feature.
"Set welcome text" pin too, maybe?
